docs/sources/alerting/fundamentals/notifications/index.md

```diff
@@ -11,19 +12,35 @@ keywords:
+Notification are sent when an alert is firing or has been resolved, and configuring how and where a notification should be sent is done via notification policies. With notification policies it is possible to configure how often a notification should be sent; and whether alerts should all be sent in the same notification, sent in grouped notifications based on a set of labels, or as seperate notifications.
 
-You can also use message templating to customize notification messages for contact point types.
+## Notification policies
 
-## Supported contact point types
+Notification policies control when and where notifications are sent. A notification policy can choose to send all alerts together in the same notification, send alerts in grouped notifications based on a set of labels, or send alerts as separate notifications. Each notification policy can have its own timings configuration that control how often notifications should be sent; and can reference one or more mute timings to inhibit notifications at certain times of the day and on certain days of the week.
 
-The following table lists the contact point types supported by Grafana.
+Notification policies are organized in a tree structure where at the root of the tree there is a notification policy called the root policy. There can be only one root policy, and the root policy cannot be deleted. The root policy also happens to be where all firing and resolved alerts are sent.
+
+Specific routing policies are children of the root policy, and can be used to match either all alerts, or a subset of alerts based on a set of labels. An example of a specific routing policy could be sending high priority alerts to Pagerduty and sending low priority alerts as emails. Another example of a specific routing policy could be sending infrastructure alerts to the Ops team, and all other alerts to another team.
+
+When an alert is sent to the root policy it looks at each specific routing policy and sends the alert to the first specific routing policy that matches the alert. If there are no specific routing policies, or none which match the alert, then the root policy is the matching policy.
+
+More information on how to configure notification policies can be found [here]({{< relref "../../manage-notifications/create-notification-policy/" >}}).
+
+## Contact points
+
+Contact points contain the configuration for sending notifications. A contact point is a list of integrations each of which sends a notification to a particular email address, service or URL. Contact points can have multiple integrations of the same kind, or a combination of integrations of different kinds. For example, a contact point could contain a Pagerduty integration; an email and Slack integration; or a Pagerduty integration, a Slack integration, and two email integrations. A contact point can also be configured with no integrations in which case no notifications are sent.
```

Comment:
Should it be `matching alerts`?

Reply:
I originally wrote that, but went back on it because technically an alert is not sent, a notification is sent. But it probably doesn't matter and alerts sounds fine 😀

Reply:
But we send an alert to the notification policy, instead of a notification right? The notification policy uses the contact point to turn that alert into a notification.